### PR TITLE
Add name field for pattern request to avoid "MesheryGeneratedPatternFile"

### PR DESCRIPTION
**Pull request by Revolyssup**
Signed-off-by: ashish <ashishjaitiwari15112000@gmail.com>

**Description**

This PR fixes #

**Notes for Reviewers**

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by Revolyssup** [comment]
@Abhishek-kumar09  Can you add "Name" field in the POST request sent from front-end?

File: models/pattern/core/pattern.go [hunk 1]
```diff
@@ -199,7 +199,7 @@ func NewPatternFileFromCytoscapeJSJSON(byt []byte) (Pattern, error) {
 
 	// Convert cytoscape struct to patternfile
 	pf := Pattern{
-		Name:     "MesheryGeneratedPatternFile",
+		Name:     name,
```
Comment (Abhishek-kumar09) [inline models/pattern/core/pattern.go:202]: Don't we need a fallback in case the name is missed?

Comment (leecalcote) [reply models/pattern/core/pattern.go:202]: Yes, we do.

**Review by leecalcote — APPROVED** [review]
Good deal.

**Pull request merged**